pkg/compute/executor.go

```diff
@@ -216,7 +221,13 @@ func (e *BaseExecutor) Start(ctx context.Context, execution *models.Execution) (
 		return
 	}
 
-	args, cleanup, err := PrepareRunArguments(ctx, e.Storages, execution, resultFolder)
+	executionStorage := filepath.Join(e.storageDirectory, fmt.Sprintf("%s/%s", execution.JobID, execution.ID))
+	if err := os.MkdirAll(executionStorage, os.ModePerm); err != nil {
```

Comment:
This sets 777 permissions, is that what we want? Would we prefer 755 or 500?

Reply:
Have set to 0755 and a local constant.  I've left the exec bit for now, meaning users can download a binary and run it if they wish (within the container theyre in).  Is this something we want to allow?